Trimble.Modus.Components/Controls/BorderlessEntry/BorderlessEntry.cs

```diff
@@ -2,7 +2,7 @@ namespace Trimble.Modus.Components;
 
 public class BorderlessEntry : Entry
 {
-    internal BorderlessEntry()

```

Comment:
Any reason for making this `public`? We don't want to expose this class to the outside world, right?